Sottodirectory in comune: mutt-1.5.24.orig/contrib e mutt-1.5.24/contrib
Sottodirectory in comune: mutt-1.5.24.orig/doc e mutt-1.5.24/doc
Sottodirectory in comune: mutt-1.5.24.orig/imap e mutt-1.5.24/imap
Sottodirectory in comune: mutt-1.5.24.orig/intl e mutt-1.5.24/intl
Sottodirectory in comune: mutt-1.5.24.orig/m4 e mutt-1.5.24/m4
Sottodirectory in comune: mutt-1.5.24.orig/po e mutt-1.5.24/po
diff -Naup mutt-1.5.24.orig/sidebar.c mutt-1.5.24/sidebar.c
--- mutt-1.5.24.orig/sidebar.c	2015-10-15 12:33:36.465539040 +0200
+++ mutt-1.5.24/sidebar.c	2015-10-15 12:35:00.051494072 +0200
@@ -310,11 +310,11 @@ int draw_sidebar(int menu) {
 		/* calculate depth of current folder and generate its display name with indented spaces */
 		int sidebar_folder_depth = 0;
 		char *sidebar_folder_name;
-		sidebar_folder_name = option(OPTSIDEBARSHORTPATH) ? mutt_basename(tmp->path) : tmp->path + maildir_is_prefix*(strlen(Maildir) + 1);
+		sidebar_folder_name = option(OPTSIDEBARSHORTPATH) ? mutt_basename(tmp->path) : tmp->path + maildir_is_prefix*(strlen(Maildir));
 		if ( maildir_is_prefix && option(OPTSIDEBARFOLDERINDENT) ) {
 			char *tmp_folder_name;
 			int i;
-			tmp_folder_name = tmp->path + strlen(Maildir) + 1;
+			tmp_folder_name = tmp->path + strlen(Maildir);
 			for (i = 0; i < strlen(tmp->path) - strlen(Maildir); i++) {
  				if (tmp_folder_name[i] == '/'  || tmp_folder_name[i] == '.') sidebar_folder_depth++;
 			}   
@@ -327,7 +327,7 @@ int draw_sidebar(int menu) {
 						tmp_folder_name++;
  				}
  				else
- 					tmp_folder_name = tmp->path + strlen(Maildir) + 1;
+ 					tmp_folder_name = tmp->path + strlen(Maildir);
  				sidebar_folder_name = malloc(strlen(tmp_folder_name) + sidebar_folder_depth*strlen(NONULL(SidebarIndentStr)) + 1);
 				sidebar_folder_name[0]=0;
 				for (i=0; i < sidebar_folder_depth; i++)
